docs/end-to-end-demo.md

```diff
@@ -58,11 +58,6 @@ carry out one or the other, then continue on with the rest of the steps.
     ```sh
     alias kubectl='microk8s kubectl'
     ```
-1. Install Helm.
-    ```sh
-    sudo apt install -y curl
-    curl -L https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 | bash
-    ```
 1. Enable Helm for MicroK8s.
```

Comment:
I love you how installed all the MicroK8s components into one step in your [tutorial](https://pretired.dazwilkin.com/posts/201022/). Could you to the same here, creating one "Enable DNS, Helm, and RBAC for MicroK8s" step with `microk8s enable dns helm3 rbac` as the command?

Reply:
Fixed: https://github.com/deislabs/akri/pull/55/commits/89e6066922a74010767586c170839667f3f4be7f